request/executor.go

```diff
@@ -6,45 +6,96 @@ import (
 	"net/http"
 )
 
+const maxRedirect = 10
```

Comment:
should this be called `maxRedirectCount`? It's called `redirectCount` below...

Reply:
Also good point.